build_projects/shared-build-targets-utils/Utils/HostVersion.cs

```diff
@@ -63,19 +63,23 @@ public override string ToString()
         // Full versions and package information.
         public string LatestHostBuildMajor => CommitCountString;
         public string LatestHostBuildMinor => "00";
-        public bool EnsureStableVersion => false;
+        public bool EnsureStableVersion => true;
 
-        // Comment below lines when stabilizing 1.1.1
-        public VerInfo LatestHostVersion => new VerInfo(1, 1, 1, ReleaseSuffix, LatestHostBuildMajor, LatestHostBuildMinor, CommitCountString);
-        public VerInfo LatestHostFxrVersion => new VerInfo(1, 1, 1, ReleaseSuffix, LatestHostBuildMajor, LatestHostBuildMinor, CommitCountString);
-        public VerInfo LatestHostPolicyVersion => new VerInfo(1, 1, 1, ReleaseSuffix, LatestHostBuildMajor, LatestHostBuildMinor, CommitCountString);
+        // Comment below lines when stabilizing 1.1.X and we are going to update one (or more) of the host packages.
+        //
+        // public VerInfo LatestHostVersion => new VerInfo(1, 1, 1, ReleaseSuffix, LatestHostBuildMajor, LatestHostBuildMinor, CommitCountString);
+        // public VerInfo LatestHostFxrVersion => new VerInfo(1, 1, 1, ReleaseSuffix, LatestHostBuildMajor, LatestHostBuildMinor, CommitCountString);
+        // public VerInfo LatestHostPolicyVersion => new VerInfo(1, 1, 1, ReleaseSuffix, LatestHostBuildMajor, LatestHostBuildMinor, CommitCountString);
+
+        // These are the versions used by GenerateMSbuildPropsFile to generate version.props that is used for 
+        // versioning of the host packages.
+        //
+        // These should only be incremented in a servicing release if we are updating one (or more) of the host packages.
+
+        public VerInfo LatestHostVersion => new VerInfo(1, 1, 0, "", "", "", CommitCountString);
```

Comment:
No, we want this to be incremented only if we service the package. Since we have not, it remains the same.